packages/bridge/README.md

```
@@ -1 +1,671 @@
+}
+```
+
+## API Reference
+
+### Native API
+
+#### `usePostMessageBridge<ReqType, ResType>()`
+
+웹뷰로 메시지를 전송하는 React Hook입니다.
+
+**타입 파라미터**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)